src/main/java/edu/cornell/kfs/pdp/document/authorization/CuPayeeACHAccountMaintenanceDocumentAuthorizer.java

```diff
@@ -0,0 +1,46 @@
+package edu.cornell.kfs.pdp.document.authorization;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.kuali.kfs.kim.api.identity.Person;
+import org.kuali.kfs.kns.service.KNSServiceLocator;
+import org.kuali.kfs.krad.document.Document;
+import org.kuali.kfs.krad.document.DocumentPresentationController;
+import org.kuali.kfs.krad.util.KRADConstants;
+import org.kuali.kfs.pdp.document.authorization.PayeeACHAccountMaintenanceDocumentAuthorizer;
+import org.kuali.kfs.sys.KFSConstants;
+import java.util.Set;
+
+public class CuPayeeACHAccountMaintenanceDocumentAuthorizer extends PayeeACHAccountMaintenanceDocumentAuthorizer {
+    private static final Logger LOG = LogManager.getLogger();
+
+    @Override
+    public boolean canApprove(Document document, Person user) {
+
+        try {
+            if (StringUtils.equalsIgnoreCase(user.getPrincipalId(), KFSConstants.SYSTEM_USER)) {
+                return true;
+            } else if (canCurrentUserBlanketApprove(document)) {
```

Comment:
In addition to always allowing blanket approvers to approve the document, you may also want to allow superusers to do so, as per the requirements from the user story. (You can double-check with the functionals to confirm which sets of initiators should still be allowed to approve these docs.) The service returned by KEWServiceLocator.getDocumentTypeService() should have some methods to help determine whether a given user is a superuser for the document.